### ISSUE TITLE
[🐛] Dependency on @backstage/core not listed

### ISSUE THREAD

**Issue by Crevil** [snyk-tech-services/backstage-plugin-snyk#24]
- `node -v`: v14.16.0
- `npm -v`: 6.14.11
- OS: (e.g. OSX, Linux, Windows, ...) Alpine 3.13

### Expected behaviour

Installing plugin on a backstage application that has migrated to the new `@backstage/core-*` packages works.

### Actual behaviour

`yarn` fails as the `@backstage/core` dependency is not listed as a dependency on this module but is required, eg. in [plugin.ts](https://github.com/snyk-tech-services/backstage-plugin-snyk/blob/develop/src/plugin.ts#L22)

```
$ yarn build
...
/src/node_modules/backstage-plugin-snyk/dist/plugin.js
Module not found: Can't resolve '@backstage/core' in '/src/node_modules/backstage-plugin-snyk/dist'
```

### Steps to reproduce

Create a backstage application that does not depend directly on `@backstage/core`, ie. have no reference to this package in `package.json`.

Run `yarn build`.

### Possible fix

By running the `@backstage/codemods` tool we might be able to update the imports accordingly.

```
npx @backstage/codemods apply core-imports src
```

**Comment by Crevil** [comment]
The codemod seems to do the trick. I'll create a PR.

**Comment by aarlaud** [comment]
Thanks for the PR! I'll review a couple of things and will get this in ASAP. thanks for the contribution! Backstage is still moving quite a bit, hard to follow the changes haha
